.github/actions/docker/action.yaml

```diff
@@ -21,7 +21,7 @@ inputs:
     description: Push tags to registry or digest only
     default: "false"
   target:
-    description: "Target stage within Dockerfile: dapi, drive-abci, dashmate-helper, test-suite"
+    description: "Target stage within Dockerfile: drive-abci, rs-dapi, dashmate-helper, test-suite"
```

Comment:
Let's call it just dapi?

Reply:
maybe later? right now it could still be confusing.